viz-lib/src/visualizations/chart/Editor/GeneralSettings.jsx

```diff
@@ -3,6 +3,8 @@ import React, { useMemo } from "react";
 import { Section, Select, Checkbox, InputNumber } from "@/components/visualizations/editor";
 import { UpdateOptionsStrategy } from "@/components/visualizations/editor/createTabbedEditor";
 import { EditorPropTypes } from "@/visualizations/prop-types";
+import * as Grid from "antd/lib/grid";
+import { cleanNumber } from "../plotly/utils";
```

Comment:
Looks that this import is not used - let's remove it if so

Reply:
Grid is used, and I would like to keep cleanNumber for `defaultValue={cleanNumber(options.rangeZMin)}`. If I use toNumber instead, the Editor shows "NaN" as default value instead of "Auto".

Reply:
Ah, I didn't notice that 🤦‍♂️ Sorry, and thank you for explaining